api/src/main/java/org/openmrs/api/db/hibernate/HibernateConceptDAO.java

```diff
@@ -2150,16 +2155,17 @@ public Drug getDrugByMapping(String code, ConceptSource conceptSource,
 								 Collection<ConceptMapType> withAnyOfTheseTypesOrOrderOfPreference) throws DAOException {
 		Session session = sessionFactory.getCurrentSession();
 		CriteriaBuilder cb = session.getCriteriaBuilder();
-		CriteriaQuery<Drug> cq = cb.createQuery(Drug.class);
```

Comment:
Could you remind me as to why we removed this from here and duplicated it down below?